package.json

```diff
@@ -71,21 +71,21 @@
     "@commitlint/config-conventional": "12.1.4",
     "@commitlint/travis-cli": "12.1.4",
     "@types/eslint": "7.2.13",
-    "@types/node": "14.17.3",
+    "@types/node": "15.14.0",
```

Comment:
I'm not sure how versioning for this works but as long as type checking works we're good, I guess.